From dcc52ea50ddeba0c8902ee22d7f984dd01bd3b59 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 19:23:13 +0800
Subject: [PATCH] video/rockchip/rk_fb: fix compile warning

drivers/video/rockchip/rk_fb.c:4002:4: warning: this else
clause does not guard... [-Wmisleading-indentation]

Change-Id: I4c75dea72a2ebde8f1271334f91d5fd890274017
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/video/rockchip/rk_fb.c | 42 +++++++++++++++++-----------------
 1 file changed, 21 insertions(+), 21 deletions(-)

diff --git a/drivers/video/rockchip/rk_fb.c b/drivers/video/rockchip/rk_fb.c
index b0b0878d44e6..45b9569d7ed7 100644
--- a/drivers/video/rockchip/rk_fb.c
+++ b/drivers/video/rockchip/rk_fb.c
@@ -4002,27 +4002,27 @@ static int rk_fb_alloc_buffer(struct fb_info *fbi)
 			else
 				fb_par->ion_hdl =
 				dev_drv_prmry->win[win_id_prmry]->area[0].ion_hdl;
-				fbi->screen_base =
-					ion_map_kernel(rk_fb->ion_client,
-						       fb_par->ion_hdl);
-				dev_drv->win[win_id]->area[0].ion_hdl =
-					fb_par->ion_hdl;
-				if (dev_drv->mmu_dev)
-					ret = ion_map_iommu(dev_drv->dev,
-							    rk_fb->ion_client,
-							    fb_par->ion_hdl,
-							    (unsigned long *)&phy_addr,
-							    (unsigned long *)&len);
-				else
-					ret = ion_phys(rk_fb->ion_client,
-						       fb_par->ion_hdl,
-						       &phy_addr, &len);
-				if (ret < 0) {
-					dev_err(fbi->dev, "ion map to get phy addr failed\n");
-					return -ENOMEM;
-				}
-				fbi->fix.smem_start = phy_addr;
-				fbi->fix.smem_len = len;
+			fbi->screen_base =
+				ion_map_kernel(rk_fb->ion_client,
+					       fb_par->ion_hdl);
+			dev_drv->win[win_id]->area[0].ion_hdl =
+				fb_par->ion_hdl;
+			if (dev_drv->mmu_dev)
+				ret = ion_map_iommu(dev_drv->dev,
+						    rk_fb->ion_client,
+						    fb_par->ion_hdl,
+						    (unsigned long *)&phy_addr,
+						    (unsigned long *)&len);
+			else
+				ret = ion_phys(rk_fb->ion_client,
+					       fb_par->ion_hdl,
+					       &phy_addr, &len);
+			if (ret < 0) {
+				dev_err(fbi->dev, "ion map to get phy addr failed\n");
+				return -ENOMEM;
+			}
+			fbi->fix.smem_start = phy_addr;
+			fbi->fix.smem_len = len;
 #else
 			fb_mem_virt = dma_alloc_writecombine(fbi->dev,
 							     fb_mem_size,
-- 
2.35.3

